apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -23,6 +27,13 @@ import { STORE_KEY as NAV_SIDEBAR_STORE_KEY } from '../../../../editing-toolkit/
  */
 import { inIframe, isEditorReadyWithBlocks, sendMessage, getPages } from '../../utils';
 
+/**
+ * Conditional dependency.  We cannot use the standard 'import' since this package is
+ * not available in the post editor and causes WSOD in that case.  Instead, we can
+ * define it from 'require' and conditionally check if it is available for use.
+ */
+const editSitePackage = require( '@wordpress/edit-site' );
```

Comment:
What a curious problem 🤔 -- this makes sense, and I can't think of a better way to handle the issue.

I do wonder if there is an established convention for this. I'll try and do a cursory search through the codebase sometime tomorrow.

Reply:
Yeah it is a bit odd.  There is some 'magic' going on in how we set up the dependency system for `wp.stuff` globals in the editor.  If you look in the post editor console in the post.php context, or the site editor in (i think index.php) you can find the dependencies available under global `wp`.  So `wp.editPost` in the post editor, or `wp.editSite` in the site editor.

The other option is digging into this magic and ensuring the post editor has `wp.editSite` as well, but to be honest I think that wouldn't be a great solution since we would be bundling an entire extra package that would never be used into the post editor, inflating the size only for the sake of how we write one odd import.  

I don't think we have imported anything from edit-site in calypso in the past, and the only reason it seems sticky here is because that dependency global magic in the editor.  In other places where package bundling works 'normally' it wouldn't be an issue.

Reply:
I did a really brief search and couldn't find any existing examples of people doing this in Calypso. This lgtm 👍